js/classes.js

```diff
@@ -301,14 +306,18 @@ class ModuleOperation extends Job {
 }
 
 class GridStrength extends Task{
+    constructor(baseData) {
+        super(baseData);
+    }
+
     collectEffects(){}
 
     getXpGain() {
         return applyMultipliers(getEnergyGeneration(), this.xpMultipliers);
     }
 
     getGridStrength(){
-         return this.getEffect(EffectType.GridStrength) + Math.floor(this.maxLevel / 10);
+         return this.level + Math.floor(this.maxLevel / 10);

```

Comment:
wie kommst du hier auf `this.maxLevel / 10` ? Das ist doch "nur" für die generated Energy relevant